srcpkgs/sdcc/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'sdcc'
 pkgname=sdcc
-version=3.9.0
+version=4.0.0
 revision=1
 archs="i686 x86_64*"
 build_style=gnu-configure
-hostmakedepends="flex bison gputils"
+hostmakedepends="automake flex bison gputils texinfo"
 makedepends="boost-devel zlib-devel"
 short_desc="Retargettable ANSI C compiler"
-maintainer="Leandro Vital <leavitals@gmail.com>"
+maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="GPL-2.0-or-later"
 homepage="http://sdcc.sourceforge.net/"
 distfiles="${SOURCEFORGE_SITE}/sdcc/${pkgname}-src-${version}.tar.bz2"
-checksum=94ecae73faf7f3feee307f89dfe3cef2d7866293c7909ea05b3b33c88d67c036
 python_version=2 #unverified
```

Comment:
Please move to `python_version=3`. According to the [SDCC Website](http://sdcc.sourceforge.net), "[a]ll Python code is now fully compatible with both Python 2.7 and Python 3.6, so Python 3 can be used instead of Python 2" as of SDCC 3.8. It looks like the only Python script installed by the package is `usr/bin/as2gbmap`, which certainly seems to be compatible with py3.

Reply:
Thanks!